go.mod

```diff
@@ -4,9 +4,12 @@ go 1.14
 
 require (
 	github.com/ajg/form v0.0.0-20160822230020-523a5da1a92f // indirect
+	github.com/bflad/tfproviderlint v0.22.0
 	github.com/fastly/go-fastly/v3 v3.3.0
 	github.com/google/go-cmp v0.5.2
+	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/terraform-plugin-docs v0.4.0
-	github.com/hashicorp/terraform-plugin-sdk v1.1.0
+	github.com/hashicorp/terraform-plugin-sdk/v2 v2.4.4
+	github.com/hashicorp/tf-sdk-migrator v1.4.0
```

Comment:
Do we need this dependency committed?

My understanding is that it's a CLI tool to automate the migration between the legacy and v2 SDKs and so once the code has been changed the dependency is no longer needed.